mail-client/alot/alot-0.8.1.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Experimental terminal UI for net-mail/notmuch written in Python"
+HOMEPAGE="https://github.com/pazz/alot"
+# SRC_URI="https://github.com/pazz/alot/tarball/${PV} -> ${P}.tar.gz"
+SRC_URI="https://github.com/pazz/alot/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( dev-python/mock[${PYTHON_USEDEP}] )
+	"
+RDEPEND="
+	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	>=app-crypt/gpgme-1.9.0[python,${PYTHON_USEDEP}]
```

Comment:
This line should be before `dev-python/configobj` (It should be sorted).

Reply:
ok

Reply:
Technically we're sorting by cat/pn (ignoring `>=`) but that's a minor.